configs/config-combinations/ubuntu16_04.yaml

```diff
@@ -0,0 +1,3 @@
+bazel: "0.21.0"
```

Comment:
Should this file have the name of the container (or some other info about the container that this config is for)?

Reply:
No that information is specified in the dependency spec file used to query the GCR resources to query. This file is being used to store the results of that query and determine when the dependency query is returning different results indicating a new version is available.

The dependency update service is aware of the concept that the Bazel version and clang version are being used to track configs. Changing the "digest" in this file to be "version" which is an opaque version string to users but internally the dependency update service will store the clang version here.

Reply:
sgtm